easybuild/easyconfigs/b/Bowtie/Bowtie-1.1.2-intel-2017a.eb

```diff
@@ -0,0 +1,20 @@
+# Modified from existing version by:
+# Robert Schmidt <rjeschmi@gmail.com>
+# Ottawa Hospital Research Institute - Bioinformatics Team
+name = 'Bowtie'
+version = '1.1.2'
+
+homepage = 'http://bowtie-bio.sourceforge.net/index.shtml'
+description = """Bowtie is an ultrafast, memory-efficient short read aligner.
+It aligns short DNA sequences (reads) to the human genome.
+"""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = ['%(namelower)s-%(version)s-src.zip']
+source_urls = ['http://download.sourceforge.net/bowtie-bio/']
+
+patches = ['int64typedef.patch','void2int.patch']
```

Comment:
@masaofujinaga please change to use one per line, and if the new patch is specific to the Bowtie version, rename it to something like `Bowtie-1.1.2_void2int.patch`

```python
patches = [
    'int64typedef.patch',
    'Bowtie-%(version)s_void2int.patch',
]
```